beep/tests/test_structure_cyclerpaths.py

```diff
@@ -0,0 +1,374 @@
+# Copyright [2020] [Toyota Research Institute]
+#
```

Comment:
I was thinking more along the lines of having these be their own files (i.e., maccor.py is tested by test_maccor_datapath.py or similar), but it doesn't really matter

Reply:
@ardunn Yeah, I guess my thought was that when you are adding a new datapath you will probably want to create the same set of tests as exist for all of the other paths. Having the tests in the same file might make that a little easier, but I could be wrong. What's your take?

Reply:
Yes that is a good point. For the time being the code looks good to me, especially since most of the datapaths only need to implement and test one method (from_file), so the file is pretty short.